cpp/iedriver/BrowserFactory.cpp

```diff
@@ -1393,4 +1435,50 @@ bool BrowserFactory::IsEdgeMode() const {
   return this->edge_ie_mode_;
 }
 
+// delete a folder recursively
+int DeleteDirectory(const std::wstring &dir_name) {
+  WIN32_FIND_DATA file_info;      
+
+  std::wstring file_pattern = dir_name + L"\\*.*";
+  HANDLE hFile = ::FindFirstFileW(file_pattern.c_str(), &file_info);
```

Comment:
Variable naming: Do not use Hungarian notation. Use `file_handle` instead.

Reply:
updated variable names to lower case with underscores.